src/mtcp/mtcp_restart.c

```diff
@@ -207,7 +207,7 @@ MTCP_PRINTF("Attach for debugging.");
   while (argc > 0) {
     // Flags for standalone debugging
     if (argc == 1) {
-      MTCP_PRINTF("Considering '%s' as a ckpt image.\n", argv[0]);
+      mtcp_printf("Considering '%s' as a ckpt image.\n", argv[0]);
```

Comment:
Why this change? We decided to use MTCP_PRINTF and DPRINTF exclusively, unless we are printing some raw data where we don't want to print line/function information with every line.